packages/plugin-import/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@modern-js/swc-plugin-import",
```

Comment:
It's seems only transform `swc-plugin-import` and `react-const-elements` into wasm plugin.
So what about others plugin?
Would add others wasm plugin in others plugin?

Reply:
Maybe in the future, as I currently working on something else, so only make those two for now